README.md

```diff
@@ -0,0 +1,172 @@
+# react-native-images-to-native-images
+
+## Introduction
+
+This react-native module will help you to convert your images to Png/Jpg/VectorDrawable (for Android)/Pdf (for iOS) in every scale.
+
+Your images must be in svg type if you want to convert them in Png/VectorDrawable/Pdf.
+If you want to convert in Jpg, your images must be a Jpg (this module will just resize your image for every scale)
+
+To show Vectors Drawables, the module need : @klarna/react-native-vector-drawable in your project dependencies, that's why we will ask you to install with our module.

```

Comment:
Pour qu'on comprenne vite, je trouve ça mieux si on peut faire des phrases plus courtes pour les 2 dernieres phrases. Par exemple: 

"
The lib can handles 2 images types in entry:
- svg -> can be converted to PNG or a mobile supported vectorial format (PDF on ios, VectorDrawable on Android). If you want the verctorial format you will need need to install the `@klarna/react-native-vector-drawable` for it)
- JPG -> kept in JPG
"